### PR TITLE
Proxy tests

### PR DESCRIPTION
**Pull request by deepak1556**

### CODE REVIEW

File: test/auth.js [hunk 1]
```diff
@@ -27,6 +29,13 @@ describe('auth', function () {
     })
   })
 
+  it('should start proxy server', function (done) {
+    server.listen(4206, function (err) {
```
Comment (jonathanong) [inline test/auth.js]: could just do `server.listen(4206, done)`

**Comment by jonathanong** [comment]
looks good. if you don't want to clean up those listeners i can just merge :)

**Comment by deepak1556** [comment]
done :+1:

**Comment by jonathanong** [comment]
thanks!

**Pull request merged**